### ISSUE TITLE
Can't build the project

### ISSUE THREAD

**Issue by matiaslina** [soumith/torch-android#33]
Hello, 

I've run into an error trying to build this project on my system, here's the [output](https://github.com/soumith/torch-android/files/198470/build-output.txt) of `build.sh`.

I've followed all the steps on the readme, the only exception was the installation of `torch` (instructions [here](http://torch.ch/docs/getting-started.html)), which was not detailed in the README instructions, but the script requested the instalation,

**Comment by borisfom** [comment]
Thanks for the report! It failed to find CUDA. 
If you don't plan to use CUDA, change -DWITH_CUDA=ON to -DWITH_CUDA=OFF in build.sh.

**Comment by matiaslina** [comment]
That was the first thing that I tried (and compiles just fine), but I
need to use it with CUDA. I've tried to define CUDA_TOOLKIT_ROOT_DIR
CUDA_NVCC_EXECUTABLE CUDA_INCLUDE_DIRS CUDA_CUDART_LIBRARY on the
cmake command (in build.sh) but it fail as well (I can post the output
tomorrow if you want).

Could it be a bug in the FindCUDA.cmake?

2016-03-31 22:13 GMT-03:00, Boris Fomitchev notifications@github.com:
